.github/workflows/ci.yml

```
@@ -25,7 +37,7 @@ jobs:
       - name: Use Node.js
         uses: actions/setup-node@v1
         with:
-          node-version: 18
+          node-version: ${{ inputs.node_version }}
```

Only use the override when the workflow is run manually, otherwise use Node 18:

```suggestion
          node-version: ${{ inputs.node_version || 18 }}
```